### DISCUSSION TITLE
RE2  modding Ada assists Leon from start to the end in biorand ?

### DISCUSSION THREAD

**Discussion by otamega72** [IntelOrca/biorand#692] [General]
> ``Ada will assist you in the streets and police station up until you meet her. She is not in every room but she's in a lot of them. Then she will also help you in some rooms near the end "

refer 1.  http://re123.bplaced.net/board/viewtopic.php?f=39&t=1029

refer 2.  https://www.youtube.com/watch?v=DLVU30hh-hA&t=781s

**Comment by IntelOrca** [comment]
I had to remove the partner in some rooms when adding enemies because they crashed the room. For example the exit of the cable car in the marshalling yard. When I had enemies and partner in that room, the game crashes - never got to the bottom of it.

Although with these new events I have found out that all NPCs must preceed enemies in the SCD to prevent crashes so that might be part of it.
